sdk-scheme-adapter/v2_0_0/backend/openapi.yaml

```diff
@@ -1224,6 +1222,8 @@ components:
           $ref: '#/components/schemas/transactionType'
         subScenario:
           $ref: '../components/schemas/TransactionSubScenario.yaml'
+        authenticationType:
+          $ref: '../components/schemas/AuthenticationType.yaml'
```

Comment:
Missing `homeR2PTransactionId`

Reply:
I think not here. Its the backend API call to core connector from SDK.